agile-app/src/app/map/page.tsx

```diff
@@ -1,7 +1,6 @@
 "use client"
 
 import {useEffect, useRef, useState} from "react";
-import {fetchMunicipalityData, fetchRegionData} from "@/scripts/geoFetching";
 import {GeoJSON, GeoJSONProps, MapContainer, TileLayer} from 'react-leaflet'
```

Comment:
Remove unused import

```suggestion
import {GeoJSON, GeoJSONProps, MapContainer, TileLayer} from 'react-leaflet'
``` -> ```suggestion
import {GeoJSON, MapContainer, TileLayer} from 'react-leaflet'
```